packages/design-system/src/components/Tabs/docs/Tabs.stories.mdx

```diff
@@ -3,7 +3,7 @@ import { FigmaIframe, FigmaImage, Use } from '~docs';
 
 import Tabs from '../';
 import { Icon } from '../../Icon';
-import Tag from '../../Tag';
+import Tag, { TagInformation } from '../../Tag';
```

Comment:
If we leave Tag as a default export, it sort of means we want to make it the default import. I don't think it's the best idea: Tag, being the `variant` picker, is also the most likely to break. Basically any chance to a variant can in theory break `Tag`. If you have only imported the variant that you need, you are safe (which is the point of making specialized components).